SampleApps/SampleApps/SampleEnclave/my_exports.cpp

```diff
@@ -283,6 +332,253 @@ HRESULT VbsEnclave::Trusted::Implementation::RunTaskpoolExample(_In_ const std::
     return S_OK;
 }
 
+//
+// User bound encryption key
+//
+HRESULT VbsEnclave::Trusted::Implementation::MyEnclaveCreateUserBoundKey(
+    _In_ const std::wstring& helloKeyName,
+    _In_ const std::wstring& pinMessage,
+    _In_ const uintptr_t windowId,
+    _In_ const uint32_t keyCredentialCacheOption,
+    _Out_ std::vector<std::uint8_t>& securedEncryptionKeyBytes)
+{
+    using namespace veil::vtl1::vtl0_functions;
+
+    try
+    {
+        debug_print(L"Start MyEnclaveCreateUserBoundKey");
+
+        // VTL1 creates secure cache configuration - VTL0 input is ignored
+        auto secureConfig = CreateSecureKeyCredentialCacheConfig();
+
+        debug_print(L"Created secure cache configuration in VTL1");
+
+        auto keyBytes = veil::vtl1::userboundkey::create_user_bound_key(
+            helloKeyName,
+            secureConfig,
+            pinMessage,
+            windowId,
+            ENCLAVE_SEALING_IDENTITY_POLICY::ENCLAVE_IDENTITY_POLICY_SEAL_SAME_IMAGE,
+            ENCLAVE_RUNTIME_POLICY_ALLOW_FULL_DEBUG,
```

Comment:
>ENCLAVE_RUNTIME_POLICY_ALLOW_FULL_DEBUG [](http://example.com/codeflow?start=12&length=39)

secure-by-default: remove this from sample or at least put it in #ifdefs so it only builds in DEBUG.

also add comment about what this specific kind of image sealing achieves